.svnignore

```diff
@@ -29,6 +31,9 @@ vendor/sirbrillig
 vendor/squizlabs
 vendor/wp-coding-standards
 vendor/automattic/jetpack-autoloader
+vendor/automattic/**/tests/
+vendor/automattic/**/README.md
+vendor/automattic/**/phpunit.xml.dist
```

Comment:
```suggestion
vendor/automattic/**/phpunit.xml
vendor/automattic/**/phpunit.xml.dist
vendor/automattic/**/composer.json
```

Reply:
This will allow us to avoid shipping at least several more files ^

Reply:
I wasn't sure about ignoring the `composer.json` files, but you're right, it's really not needed in here.